ts/features/payments/home/components/PaymentsHomeScreenHistoryList.tsx

```diff
@@ -0,0 +1,52 @@
+import { ListItemTransaction } from "@pagopa/io-app-design-system";
+import * as _ from "lodash";
+import * as React from "react";
+import TypedI18n from "../../../../i18n";
+import { IndexedById } from "../../../../store/helpers/indexer";
+import { Transaction } from "../../../../types/pagopa";
+import { getAccessibleAmountText } from "../../../../utils/accessibility";
+import { format } from "../../../../utils/dates";
+import { formatNumberCurrencyCents } from "../../../idpay/common/utils/strings";
+import { isTransactionSuccess } from "../store/selectors";
+
+export const getHistoryList = (history: IndexedById<Transaction>) => {
```

Comment:
This file does not conform to our style guides: the CamelCase file names are for components/function components, but in this file we are exporting only a function. What do you think about moving this function directly to the component which uses it?

Reply:
right, it used to be a component during a development iteration and I've forgotten to update the naming; good catch!
As for moving it to the component's file though, I believe it is a cleaner choice to give it its own file since it is a rather large function and it would compromise readability -- also it will need to be widely updated once we finally move to the new payment history structure. What do you think?